nova/virt/vmwareapi/ds_util.py

```diff
@@ -443,29 +431,37 @@ def get_sub_folders(session, ds_browser, ds_path):
     return set()
 
 
-def _filter_datastores_matching_storage_policy(session, data_stores,
+
```

Comment:
This newly-added empty line breaks pep8.

Reply:
Just saw it too. It is now fixed.